bgpd/bgp_vty.c

```diff
@@ -2577,7 +2577,7 @@ static struct peer_group *listen_range_exists(struct bgp *bgp,
 
 DEFUN (bgp_listen_range,
        bgp_listen_range_cmd,
-       "bgp listen range <A.B.C.D/M|X:X::X:X/M> peer-group WORD",
+       "bgp listen range <A.B.C.D/M|X:X::X:X/M> peer-group PGNAME",
```

Comment:
@ton31337 I see only changes to replace "WORD" with "PGNAME". Thats all ?

Reply:
yep 🙂

Reply:
Checked the output from the attached gif file. Thanks.